### PR TITLE
compliance: upgrade inspec to latest current build

### PR DESCRIPTION
**Pull request by stevendanna**
We can't merge this because of other issues at the moment, but this is to 
test the overall build.

Signed-off-by: Steven Danna <steve@chef.io>

### CODE REVIEW

**Comment by lancewf** [comment]
The PR will fix the broken buildkite tests. 
https://github.com/chef/automate/pull/4424

**Comment by alexpop** [comment]
gcp-scan

**Comment by alexpop** [comment]
Please see above a screenshot of a successful Google Cloud API scan with InSpec 4.24.32 and profile `InSpec GCP CIS 1.1 Benchmark` version `1.1.0-25`.
It shows `passed`, `skipped` and `failed` controls as expected based on the resources found in the target GCP account.

These are the steps I took to validate this new InSpec version + profile.

I created a fresh dev environment in ec2 from the `ssd/upgrade-inspec` automate branch.
I started all services with the habitat script: `start_all_services`, which pulled the latest dev habitat packages for the services, build: 20210212140448

I checked the following directory to see that an older version (`4.23.4`) of inspec is used:

```
/hab/pkgs/chef/inspec/  
```

I ran the following habitat script in the environment:
```
rebuild components/compliance-service 
```

Then I validated that a new directory has been created for the new inspec (4.24.32) under:

```
/hab/pkgs/chef/inspec/
```

I logged in to the Automate dashboard and imported this profile from Automate UI > Compliance > Profiles > Available:

`InSpec GCP CIS 1.1 Benchmark` version `1.1.0-25`

Then I went to Automate UI > Settings > Node Integrations > Create Integration > Google Cloud

For the name field, I used the project name I have in Google Cloud `apop-nlp`. For the text area below label `Enter your service account JSON credentials` I went to the service account of my Google Cloud:

https://console.cloud.google.com/iam-admin/serviceaccounts

created a service account and generated a json key for it, which I downloaded locally. This is the json content that I used for the text area I mentioned above.

Clicked `Save Integration` and proceeded to trigger a scan.
Before, I checked to make sure my Google Cloud account has the Compute API enabled.

Then I went to Automate UI > Create Scan Job > selected the `apop-nlp` node integration > Next > selected the `InSpec GCP CIS 1.1 Benchmark` profile > Next > gave the scan job a name > Save

Before the scan I also used the `sl` command in automate to see the logs, which showed the new inspec version being used for the scan.

Waited for the scan job to reach status `completed` and clicked the Report button to view the results!

**Review by alexpop — APPROVED** [review]

File: .expeditor/verify.pipeline.yml [hunk 1]
```diff
@@ -590,7 +621,7 @@ steps:
       - sudo scripts/install_golang.sh
       - sudo scripts/install_grpcurl.sh
       - cd components/compliance-service
-      - sudo -E PATH=\$(hab pkg path core/ruby)/bin:\$PATH make test-db
+      - sudo -E PATH=\$(hab pkg path core/ruby26)/bin:\$PATH make test-db
```
Comment (tas50) [inline .expeditor/verify.pipeline.yml:624]: InSpec is Ruby 2.7 now

Comment (kalroy) [reply .expeditor/verify.pipeline.yml:624]: Thanks @tas50 for letting us know. We will upgrade to ruby 2.7 in the upcoming releases. This was to handle  vulnerability for an user which we  believe can be resolved by upgrading to ruby 2.6.

**Pull request merged**